diff --git a/bin/varnishd/cache/cache_director.c b/bin/varnishd/cache/cache_director.c
index f5552fe5f..c1c60b895 100644
--- a/bin/varnishd/cache/cache_director.c
+++ b/bin/varnishd/cache/cache_director.c
@@ -76,7 +76,7 @@ VDI_Ahealth(const struct director *d)
 
 /* Resolve director --------------------------------------------------*/
 
-static VCL_BACKEND
+VCL_BACKEND
 VDI_Resolve(VRT_CTX)
 {
 	const struct director *d;
diff --git a/include/vrt.h b/include/vrt.h
index a0c142c52..d7eb04f76 100644
--- a/include/vrt.h
+++ b/include/vrt.h
@@ -461,6 +461,7 @@ VCL_BACKEND VRT_AddDirector(VRT_CTX, const struct vdi_methods *,
 void VRT_SetHealth(VCL_BACKEND d, int health);
 void VRT_DisableDirector(VCL_BACKEND);
 void VRT_DelDirector(VCL_BACKEND *);
+VCL_BACKEND VDI_Resolve(VRT_CTX);
 
 /* Suckaddr related */
 int VRT_VSA_GetPtr(const struct suckaddr *sua, const unsigned char ** dst);
